fabric-extra-tag-api-v1/build.gradle

```diff
@@ -0,0 +1,13 @@
+archivesBaseName = "fabric-registry-tag-api-v1"
```

Comment:
Is it `registry-tag-api` or `extra-tag-api`? I'll be honest, I'm really not a fan of `extra-tag-api` but unfortunately I don't really have an alternative idea. Maybe `fabric-more-tag-types-api`?

Reply:
Tbf `registry-tag-api` was the original name, but the fact we don't cover every registry makes it misleading.

The name could be improved for sure.